Dockerfile

```diff
@@ -3,6 +3,8 @@ FROM alpine:edge
 RUN apk --update add ruby ruby-dev ruby-bundler && \
     rm -fr /usr/share/ri
 
+RUN apk --update add ca-certificates
```

Comment:
More info: https://docs.docker.com/articles/dockerfile_best-practices/#minimize-the-number-of-layers